README.md

```diff
@@ -1,3 +1,100 @@
 # Launching Node
 
 ## Overview
+
+In this lesson, you'll learn how to run Node scripts with and without parameters.
+
+## Objectives
+
+1. Describe how to execute a Node script from a terminal
+1. Use the node help command to get the list of node CLI commands
+1. Use node CLI command options and flags to launch node scripts in different modes
+
+## Describe how to execute a Node script from a terminal
+
+So let's say you already wrote a Node script (even if you haven't). It can be as simple as a Hello World, for example we can use `console` interface:
```

Comment:
What's the difference between a node script and a normal js file? students will have had decent experience with js, but not with node